front/components/LP/device.tsx

```diff
@@ -8,7 +8,7 @@ export default class Device extends React.Component<DeviceProps, any> {
     return (
       <div className="device">
           <div className="heading">
-            Available anywhere.
+            Available anywhere!
```

Comment:
All sentences except title end with the period, and I think that looks better for my personal preference. So I would like to keep the period here as well.